modules/dxp/apps/search-experiences/search-experiences-rest-impl/src/main/java/com/liferay/search/experiences/rest/internal/dto/v1_0/converter/SXPElementDTOConverter.java

```diff
@@ -59,12 +59,14 @@ public SXPElement toDTO(
 			com.liferay.search.experiences.model.SXPElement sxpElement)
 		throws Exception {
 
+		String defaultLanguageId = sxpElement.getDefaultLanguageId();
```

Comment:
Hi @gustavolimav, could you inline this like the dtoConverterContext.getLocale() was?